tests/stage_2/mod-defmacro-1.txt

```diff
@@ -7,4 +7,4 @@ brun -s2 '(opt (com (q \
 		(map (lambda X (* X X)) (10 20 30 40 50)) \
 	) \
 )))'
-(q (q (100 400 900 1600 2500)))
+(q 1 100 400 900 1600 2500)
```

Comment:
`(q 1 100 400 900 1600 2500)` == `(q . (q . (100 400 900 1600 2500)))`

Reply:
I don't get this.